core/block/editor/table/table.go

```diff
@@ -1006,3 +265,38 @@ func (tb Table) Iterate(f func(b simple.Block, pos CellPosition) bool) error {
 	}
 	return nil
 }
+
+// CheckTableBlocksMove checks if Insert operation is allowed in case table blocks are affected
+func CheckTableBlocksMove(st *state.State, target string, pos model.BlockPosition, blockIds []string) (string, model.BlockPosition, error) {
+	// nolint:errcheck
+	if t, _ := NewTable(st, target); t != nil {
```

Comment:
you could just check error (err == nil && t != nil)

Reply:
Thank you! Done